simulator/infra/cache/cache_tag_array.h

```diff
@@ -50,8 +50,8 @@ class CacheTagArraySizeCheck : public Log
             uint32 addr_size_in_bits
         );
 
-        const uint32 size_in_bytes;
-        const uint32 ways;
```

Comment:
Removal of constant correctness here

Reply:
I don't understand how to deal with it. Should interface class specify these attributes as constant, and the inherited InfiniteCacheTagArray shadow them with it's own non-const ones ( or even vice versa: CacheTagArray and AlwaysHitCacheTagArray shadow non-contant attributes with the constant ones)?

Reply:
InfiniteCacheTagArray should not be inherited from CacheTagArray. They should be inherited from the common base interface that contains no implementation.

Reply:
Now, I got it. Exuse me for missunderstanding l.